test/api/vote.test.js

```diff
@@ -75,72 +78,178 @@ describe('Vote API', () => {
     passportStub.uninstall();
   });
 
-  it('should not be possible to vote with an invalid ObjectId as alternativeId', async () => {
+  it('should not be possible to vote without election', async () => {
     const { body: error } = await request(app)
       .post('/api/vote')
-      .send(votePayload('bad alternative'))
-      .expect(404)
+      .send({ priorities: [] })
```

Comment:
Could you add something here to make sure we encapsulate only the missing election?

Reply:
hmm, this don't make any sense 🤷🏻  🙃